twentytwenty-blocks/twentytwenty-styles/style.css

```diff
@@ -305,48 +306,47 @@ h6,
 .faux-heading {
 	font-feature-settings: "lnum";
 	font-variant-numeric: lining-nums;
-	font-weight: 700;
+	font-weight:  var(--wp--preset--font-weight--heading);
 	letter-spacing: -0.0415625em;
-	line-height: 1.25;
+	line-height:  var(--wp--preset--line-height--heading);
 	margin: 3.5rem 0 2rem;
 }
 
 h1,
 .heading-size-1 {
-	font-size: 3.6rem;
-	font-weight: 800;
-	line-height: 1.138888889;
+	font-size: var(--wp--preset--font-size--normal-heading-1);
```

Comment:
Where are these `font-size--normal-heading-*` variables coming from?

Reply:
Whoops. I've replaced those in cfeb4dc.